.github/workflows/claude.yml

```
@@ -2,92 +2,466 @@ name: Claude Code
+      - name: Install gh-models extension
+        if: steps.detect_model.outputs.selection_method == 'default'
+        run: gh extension install https://github.com/github/gh-models
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: AI model selection
+        if: steps.detect_model.outputs.selection_method == 'default'
+        id: ai_select
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
```

actionlint (1.7.9)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)